docusaurus/docs/reactnative/guides/performance_boost.mdx

```diff
@@ -0,0 +1,91 @@
+---
+id: performance-boost
+sidebar_position: 5
+title: Performance Boost
+---
+
+import channelSettings from '../assets/guides/performance-boost/channel_settings.jpeg'
+
+React Native Chat SDK provides feature rich components which caters to all the types of chat messaging applications.
+Although some of the features or components may come with performance cost, which you will want to avoid for the cases of high
+concurrency applications such as livestreaming. In this guide we have noted down some tips for improving the performance of your
+chat application if you are expecting heavy traffic on your chat.
+
+## Channel Settings
+
+Within each channel type, some settings are available that apply to all channels of that type.
+Among these settings are the ability to enable/disable each event type.
+When disabled, events of that type (for channels of this type) will not be passed through to a client's open WebSocket connection.
+It's also important to note that increasing the events enabled on a channel type also increases the load on clients in those channels.
+
+- For Livestream type channels, we recommend disabling Connect, Read Events, and Typing Events. These will cause performance issues and don't generally add to the user experience in these use cases.
+- Also consider using [Slow Mode](https://getstream.io/chat/docs/slow_mode/?language=js#channel-slow-mode) for Livestream events.
+
+<img src={channelSettings} />
+
+## State update throttling
+
+[Channel](../core-components/channel.mdx) component updates its internal react state based on chat related events received from websocket.
+By default these state updates are throttled to once per 500 ms. For high traffic applications, you may want to adjust throttling
+interval to a higher number. You can configure throttle interval via props on Channel component:
+
+- `newMessageStateUpdateThrottleInterval` prop adjusts the throttling interval for state updates when new messages arrive in chat.
+- `stateUpdateThrottleInterval` prop adjusts the throttling interval for state updates for all the other updates to channel except new message.
+
+We recommend following configuration, but you can adjust it as per your requirement.
```

Comment:
```suggestion
The [Channel](../core-components/channel.mdx) component updates its internal react state based on chat related events received from the websocket.	 
  
 By default, these state updates are throttled to once per 500 ms. For high-traffic applications, you may want to adjust the throttling interval to a higher number. You can configure throttle interval via the following props on the Channel component:	 
   
 - The `newMessageStateUpdateThrottleInterval` prop adjusts the throttling interval for state updates when new messages arrive in chat.	 
  
 - The `stateUpdateThrottleInterval` prop adjusts the throttling interval for state updates for all the other updates to a channel except new message.	 
  
We recommend the following configuration, but you can adjust it to fit the requirements of your application.
```

Reply:
Might also be worth mentioning the unit of measure for the values of the props? Although, I don't imagine many people will think things are throttled by 800 seconds, haha.